pom.xml

```diff
@@ -139,7 +139,6 @@
                 <groupId>joda-time</groupId>
                 <artifactId>joda-time</artifactId>
                 <version>2.2</version>
-                <scope>test</scope>
```

Comment:
Sorry @dkayiwa for not responding to your question. I have missed that one. 
I can confirm that core provides this library. 
pom.xml edited :)